ramls/schemas/GOBIMapping.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "GOBI DataMapping Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "fieldName" : {
+      "type" : "string",
+      "description": "name of the field to map"
+    },
+    "defaultValue" : {
+      "type" : "string",
+      "description": "default value of field"
+    },
+    "fieldPath" : {
```

Comment:
not a big deal, but "xpath" or "from"/"fromPath" or "source"/"sourcePath" might be a more intuitive name